service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -57,6 +57,6 @@ public HearingBooking getHearingBookingByUUID(List<Element<HearingBooking>> hear
             .filter(hearingBookingElement -> hearingBookingElement.getId().equals(elementId))
             .map(Element::getValue)
             .findFirst()
-            .orElse(HearingBooking.builder().build());
+            .orElse(null);
```

Comment:
Could return Optional empty or it's too big change on this level? Generally the code should operate on the optionals.empty rather tha nulls.

Reply:
hmm, this was brought up in @Jamesnelson117 PR. Have I just not merged correctly?

Reply:
I know this wasn't made in this PR, but something that I just found out that it is usually better (especially if referencing a resource intensive method) is to use `orElseGet` instead of `orElse` as `orElseGet` is lazy